### PR TITLE
[LENS-1324] Adds examples on getting trackingId and deviceId

### PR DESCRIPTION
**Pull request by npushkarskii**
This PR adds examples on how to:

* get the current deviceId

* get the current trackingId

README already had the example for setting the new `deviceId`.

https://coveord.atlassian.net/browse/LENS-1324

### CODE REVIEW

File: README.md
Comment (jsonDaCruzo) [inline README.md:58]:
```suggestion
Calling `restartWithCustomDeviceId(null)` restores default behaviour. Changing the device identifier once the SDK has already been started will restart the SDK. This will clear all caches and resend startup events.
```

**Review by jsonDaCruzo — COMMENTED** [review]
Thanks @npushkarskii, made a grammatical suggestion. Please wait for @hporter approval though before merging as he is a better judge on the code examples provided.

**Review by hporter — APPROVED** [review]

**Pull request merged**
